website/client/src/assets/scss/typography.scss

```diff
@@ -77,3 +77,7 @@ h4 {
 .opacity-75 {
   opacity: 0.75;
 }
+
+*:focus {
```

Comment:
this is going to be applied to every element around the website?

Reply:
@alexandreabeh after talking with @Tressley we'd like not to apply the style to all elements across the website but in the case of the purple outline to limit it to the tasks only

Reply:
Ok, will do

Reply:
@paglias and @Tressley I've submitted changes that are ready for re-review. Thanks!